src/components/Login.tsx

```diff
@@ -24,8 +24,9 @@ const uiConfig = {
         firebaseuiAuth.AnonymousAuthProvider.PROVIDER_ID,
     ],
     callbacks: {
-        // Avoid redirects after sign-in.
-        signInSuccessWithAuthResult: () => false,
+        signInSuccessWithAuthResult: ({ user }) => {
+            return !user.isAnonymous
```

Comment:
- pls consider leaving the comment since I find the method name not that clear
- is the anonymous user correctly redirected to the `Home` page ?